app/src/main/java/ar/scacchipa/twittercloneapp/repository/DbContants.kt

```diff
@@ -0,0 +1,25 @@
+package ar.scacchipa.twittercloneapp.repository
+
+object DbContants {
```

Comment:
Please change the name to something related to what are this constants for, DB implies something related to data bases and this is not the case